rm/rm-node/src/main/java/org/ow2/proactive/resourcemanager/utils/RMNodeStarter.java

```diff
@@ -130,6 +150,13 @@
     public static final String NODE_ACCESS_TOKEN = "proactive.node.access.token";
 
     /**
+     * The maximum time to wait in milliseconds for retrieving the answer
+     * for the node availability report that is pushed periodically.
+     */
+    @SuppressWarnings("squid:S1444")
+    public static int NODE_AVAILABILITY_REPORT_TIMEOUT_DELAY = 5000; // in ms
```

Comment:
would it not be better to make it a final variable and name it DEFAULT_NODE_AVAILABILITY_REPORT_TIMEOUT_DELAY , and use another variable for the configured value ?